torch/_inductor/ir.py

```diff
@@ -2939,10 +2934,11 @@ def codegen(self, wrapper):
             if self.indices[i] is not None:
                 indices.append(next(iter_valid_indices))
             else:
-                indices.append("None")
-        wrapper.writeline(
-            f"{self.kernel}({x}, [{','.join(indices)}], {values}, {repr(self.constant_args[0])})"
-        )
+                indices.append(V.graph.wrapper_code.none_str)
+
+        indices = f"{V.graph.wrapper_code.open_bracket}{', '.join(indices)}{V.graph.wrapper_code.closed_bracket}"
```

Comment:
How about add a function to make it less verbose, e.g., 
```suggestion
        indices = f"{V.graph.wrapper_code.codegen_list_init(', '.join(indices))}"
```

Reply:
I will do it in a later PR.